### PR TITLE
Add section-3

### PR DESCRIPTION
**Pull request by katsu223**
[Deploy link](https://katsu223.surge.sh)

### CODE REVIEW

File: resources/css/index.css [hunk 1]
```diff
@@ -202,3 +206,48 @@ body, html, ul, ol, li, p, h1, h2, h3, h4 {
...
   justify-content: center;
 }
+
+.second-section{
+  background:  url("../images/background/section-2.png") no-repeat;
+  padding: 100px 215px 100px 815px;
+}
+
+.second-section .content {
+  background: rgba(0, 0, 0, 0.6);
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
  background: rgba(0, 0, 0, .6);
```

File: resources/css/index.css [hunk 2]
```diff
@@ -202,3 +206,48 @@ body, html, ul, ol, li, p, h1, h2, h3, h4 {
...
+}
+
+.second-section .content .content-title {
+  color: #fff;
+  margin: 0;
+  margin-bottom: 25px;
+}
+
+.second-section .content .divider {
+  margin: 0;
```
Comment (magicsd) [inline resources/css/index.css]: Why do you need this?

File: resources/css/index.css [hunk 3]
```diff
@@ -202,3 +206,48 @@ body, html, ul, ol, li, p, h1, h2, h3, h4 {
...
+  line-height: 24px;
+  margin-top: 30px;
+}
+
+.images-section {
+  margin-top: 30px;
+}
+
+.ingreditets-image {
+  border: 5px solid rgba(255, 255, 255, 0.1);
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
  border: 5px solid rgba(255, 255, 255, .1);
```

File: resources/css/index.css [hunk 4]
```diff
@@ -202,3 +206,48 @@ body, html, ul, ol, li, p, h1, h2, h3, h4 {
...
+  color: #fff;
+  line-height: 24px;
+  margin-top: 30px;
+}
+
+.images-section {
+  margin-top: 30px;
+}
+
+.ingreditets-image {
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
.ingredients-image {
```

File: resources/css/index.css [hunk 5]
```diff
@@ -202,3 +206,48 @@ body, html, ul, ol, li, p, h1, h2, h3, h4 {
...
+  border: 5px solid rgba(255, 255, 255, 0.1);
+  border-radius: 50%;
+  margin-left: 25px;
+}
+
+.ingreditets-image:first-child {
+  margin-left: 0px;
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
  margin-left: 0;
```

File: resources/css/index.css (hunk 1 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 2 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 3 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 4 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

File: resources/css/index.css (hunk 5 above)
Comment (katsu223) [reply resources/css/index.css]: fixed.

**Pull request merged**